commcare_connect/opportunity/models.py

```diff
@@ -387,6 +387,16 @@ class VisitValidationStatus(models.TextChoices):
     trial = "trial", gettext("Trial")
 
 
+class PaymentInvoice(models.Model):
+    opportunity = models.ForeignKey(Opportunity, on_delete=models.CASCADE)
+    amount = models.PositiveIntegerField()
+    date = models.DateField()
+    invoice_number = models.CharField(max_length=50)
```

Comment:
Don't we need any unique constraints? There could be multiple invoices with same invoice-number this way, right?